libs/h3/src/test/java/org/elasticsearch/h3/ParentChildNavigationTests.java

```diff
@@ -65,4 +72,38 @@ private void assertHexRing(int res, String h3Address, String[] children) {
             assertEquals(children[i], ring[positions[i - 1]]);
         }
     }
+
+    public void testNoChildrenIntersecting() {
+        String[] h3Addresses = H3.getStringRes0Cells();
+        String h3Address = RandomPicks.randomFrom(random(), h3Addresses);
+        for (int i = 1; i <= H3.MAX_H3_RES; i++) {
```

Comment:
The for loop seems to imply the test will test at different resolutions, but it does not. It only selects 15 random level0 cells and tests those. I'm guessing you were planning something else, but did not complete it?